pom.xml

```diff
@@ -41,7 +41,7 @@
     <maven.compiler.target>1.8</maven.compiler.target>
 
     <!-- JBoss Tools Integration Stack version -->
-    <jbtis.version>4.4.1.Final</jbtis.version>
+    <jbtis.version>4.4.2.AM1</jbtis.version>
```

Comment:
The release is not based on the master, so it is the 9.1.x branch which is important for that. i created https://issues.jboss.org/browse/FUSETOOLS-2308